CHANGELOG.md

```diff
@@ -49,6 +49,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Change http code on create index API with bad input raising NotXContentException from 500 to 400 ([#4773](https://github.com/opensearch-project/OpenSearch/pull/4773))
 - Improve summary error message for invalid setting updates ([#4792](https://github.com/opensearch-project/OpenSearch/pull/4792))
 - Return 409 Conflict HTTP status instead of 503 on failure to concurrently execute snapshots ([#8986](https://github.com/opensearch-project/OpenSearch/pull/5855))
+- Add task completionCount in search_backpressure stats API
```

Comment:
nitpick: These changes are documented as part of the release notes, so treat them as user facing documentation.

```suggestion
- Add `taskCompletionCount` in `search_backpressure` stats API
```

or

```suggestion
- Add task completion count in search backpressure stats API
```

Reply:
Makes sense, Thanks!